notification-manager/src/main/resources/application-DEV.yaml

```diff
@@ -1,9 +1,13 @@
+spring:
+  kafka:
+    bootstrap-servers: localhost:9092
```

Comment:
same for payment manager

Reply:
i don't get it what should i do. How i understand, move every single port that based on localhost to DEV.yml, am i correct?

Reply:
correct. Check payment-manager application.yaml. There's bootstrap-servers: localhost:9092 which should be moved as well